dali/pipeline/data/types.h

```diff
@@ -551,6 +555,7 @@ DALI_REGISTER_TYPE(std::vector<bool>, DALI_BOOL_VEC);
 DALI_REGISTER_TYPE(std::vector<int>, DALI_INT_VEC);
 DALI_REGISTER_TYPE(std::vector<std::string>, DALI_STRING_VEC);
 DALI_REGISTER_TYPE(std::vector<float>, DALI_FLOAT_VEC);
+DALI_REGISTER_TYPE(std::vector<TensorLayout>, DALI_TENSOR_LAYOUT_VEC);
```

Comment:
Do you use this type anywhere?

Reply:
Yes. 
```c++
.AddOptionalArg<std::vector<TensorLayout>>("output_layouts", ...
```
It's needed to have a nice documentation for the argument:
![doc](https://user-images.githubusercontent.com/32324320/100074014-82120080-2e3e-11eb-8840-0243c841cded.PNG)

Reply:
ok